Code/GraphMol/CIPLabeler/Edge.cpp

```diff
@@ -0,0 +1,49 @@
+//
+//
+//  Copyright (C) 2020 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <stdexcept>
+
+#include "Edge.h"
+#include "Node.h"
+
+namespace RDKit {
+namespace CIPLabeler {
+
+Edge::Edge(Node *beg, Node *end, Bond *bond)
+    : dp_beg{beg}, dp_end{end}, dp_bond{bond} {}
+
+Node *Edge::getOther(Node *node) const {
```

Comment:
I'm not sure, but it seems like this needs a `PRECONDITION` on `node`, right?

Reply:
Definitely. I haven't been too careful with those. I have the feeling there are many places where these are missing.